### PR TITLE
Some VARCHARs replaced with TEXT + CHECK(length)

### PR DESCRIPTION
**Pull request by ciguenablanca**

### CODE REVIEW

File: src/tables.sql [hunk 1]
```diff
@@ -286,22 +303,24 @@ create table media(
     type varchar(50) not null,
     description varchar(1024),
     hash varchar(255),
-    fileName varchar(100),
+    file_name varchar(100),
```
[the remark this answers is not in the document]
Comment (ciguenablanca) [reply src/tables.sql:323]: yeah I'm correcting them)

File: src/tables.sql [hunk 2]
```diff
@@ -114,26 +114,43 @@ create table property(
...
     to_date date,
 
-    productionYear smallint,
-    registrationYear smallint,
+    production_year smallint,
+    registration_year smallint,
 
     vin varchar(50),
+
+    person_id bigint,
```
[the remark this answers is not in the document]
Comment (ciguenablanca) [reply src/tables.sql:144]: person_id is one who drives
owner_id is one who oficially ownes
typical case a car registered to wife

File: src/tables.sql [hunk 3]
```diff
@@ -114,26 +114,43 @@ create table property(
     CHECK (length(notes) <= 4096)
 );
 
+DROP TYPE IF EXISTS vehicle_type;
```
[the remark this answers is not in the document]
Comment (ciguenablanca) [reply src/tables.sql:117]: yes

File: src/tables.sql [hunk 4]
```diff
@@ -22,14 +22,15 @@ drop table if exists source;
...
+    src text not null,
+    CHECK (length(src) <= 2048)
 );
 
 
 create table person(
     id bigserial primary key,
     -- Полное имя, если не структурировано четко в ФИО
-    name varchar(255),
+    full_name text,
```
[the remark this answers is not in the document]
Comment (ciguenablanca) [reply src/tables.sql:33]: agree 255 should be fine

File: src/tables.sql (hunk 2 above)
[the remark this answers is not in the document]
Comment (anazar2001) [reply src/tables.sql:147]: was thinking about this in the context of address. Maybe we should do the same owner/person thing there as well.

**Review by anazar2001 — APPROVED** [review]

**Pull request merged**